src/main/java/dp/handler/Handler.java

```diff
@@ -122,7 +122,13 @@ public void listen(final ExportedFile message) {
         MessageType messageType = GetMessageType(message);
 
         try {
-
+            if (message.getRowCount() != null){
+                if (message.getDatasetId() != null) {
+                    info().datasetID(message.getDatasetId().toString()).rowCount(message.getRowCount().toString());
```

Comment:
Use rowCount in here. Also do the same thing for message.getDatasetId() and message.getInstanceId().